modules/navigation/nav_map.h

```diff
@@ -199,9 +201,7 @@ class NavMap : public NavRid {
 		return obstacles;
 	}
 
-	uint32_t get_map_update_id() const {
-		return map_update_id;
-	}
+	uint32_t get_map_update_id() const { return iteration_id; }
```

Comment:
is this method left for compatibility purposes?

Reply:
More to keep the changes smaller. Need to change the agent, obstacle and map that all make use of this. I can do that.

Reply:
no need - perhaps you could just leave a in-code comment that this needs method to be removed